From 9d40b6e665f67f5096eba41bea13492d3eda056c Mon Sep 17 00:00:00 2001
From: Samuli Piippo <samuli.piippo@qt.io>
Date: Fri, 24 Nov 2017 15:16:31 +0200
Subject: [PATCH] Bootstrap without linkat feature

qmake does not work together with pseudo when unnamed temporary files
are used with linkat.

Upstream-Status: Inappropriate [OE specific]
[YOCTO #11996]
---
 src/corelib/global/qconfig-bootstrapped.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/corelib/global/qconfig-bootstrapped.h b/src/corelib/global/qconfig-bootstrapped.h
index 8cfa40e461..f2fba9ebd8 100644
--- a/src/corelib/global/qconfig-bootstrapped.h
+++ b/src/corelib/global/qconfig-bootstrapped.h
@@ -91,7 +91,7 @@
 #define QT_FEATURE_itemmodel -1
 #define QT_FEATURE_library -1
 #ifdef __linux__
-# define QT_FEATURE_linkat 1
+# define QT_FEATURE_linkat -1
 #else
 # define QT_FEATURE_linkat -1
 #endif
